CMakeLists.txt

```diff
@@ -0,0 +1,54 @@
+cmake_minimum_required(VERSION 3.15)
+
+project(yc-libvhost-server)
+
+if(UNIX AND NOT APPLE)
+  set(LINUX TRUE)
+endif()
+
+if(NOT LINUX)
+  message(FATAL_ERROR "Unsupported platform")
+endif()
+
+SET(libvhost_log_verbosity "LOG_INFO" CACHE STRING "Libvhost log verbosity")
+message("Compiler ${CMAKE_C_COMPILER}")
+message("Libvhost log verbosity: ${libvhost_log_verbosity}")
```

Comment:
Why do we need these two logged?

Reply:
I think this is good to have in build logs because this is essentially two options that affect the library build.
If you find them useless I'll drop these messages.